joern-cli/frontends/jssrc2cpg/src/main/scala/io/joern/jssrc2cpg/astcreation/AstForDeclarationsCreator.scala

```diff
@@ -429,15 +443,28 @@ trait AstForDeclarationsCreator { this: AstCreator =>
     }
   }
 
-  private def createImportNodeAndAttachToAst(
+  def createImportNodeAndAttachToCall(
     impDecl: BabelNodeInfo,
     importedEntity: String,
-    importedAs: String
+    importedAs: String,
+    call: Option[NewCall]
   ): NewImport = {
-    val impNode = createImportNode(impDecl, Option(importedEntity).filter(_.trim.nonEmpty), importedAs)
-    methodAstParentStack.collectFirst { case namespaceBlockNode: NewNamespaceBlock =>
-      diffGraph.addEdge(namespaceBlockNode, impNode, EdgeTypes.AST)
-    }
+    createImportNodeAndAttachToCall(impDecl.code.stripSuffix(";"), importedEntity, importedAs, call)
+  }
+
+  def createImportNodeAndAttachToCall(
+    code: String,
+    importedEntity: String,
+    importedAs: String,
+    call: Option[NewCall]
+  ): NewImport = {
+    val impNode = NewImport()
+      .code(code)
+      .importedEntity(importedEntity)
+      .importedAs(importedAs)
+      .lineNumber(call.flatMap(_.lineNumber))
+      .columnNumber(call.flatMap(_.lineNumber))
```

Comment:
Should this not be:

```suggestion
      .columnNumber(call.flatMap(_.columnNumber))
```

Reply:
You are right, missed that one in my review :-(
This needs to be fixed.